preset.py

```diff
@@ -3,144 +3,144 @@
 def place(board, shape, a, b):
     """Places a preset shape."""
     if shape == 1:
-        if a + 2 >= config.Width + config.Cushion:
-            a = config.Width + config.Cushion - 3
-        if b + 2 >= config.Height + config.Cushion:
-            b = config.Height + config.Cushion - 3
-        board[a][b].kill()
-        board[a + 1][b].birth(config.Square)
-        board[a + 2][b].kill()
-        board[a][b + 1].kill()
-        board[a + 1][b + 1].kill()
-        board[a + 2][b + 1].birth(config.Square)
-        board[a][b + 2].birth(config.Square)
-        board[a + 1][b + 2].birth(config.Square)
-        board[a + 2][b + 2].birth(config.Square)
+        if a + 2 >= board.Width + board.Cushion:
+            a = board.Width + board.Cushion - 3
+        if b + 2 >= board.Height + board.Cushion:
+            b = board.Height + board.Cushion - 3
+        board.Cell[a][b].kill()
+        board.Cell[a + 1][b].birth(config.Square)
+        board.Cell[a + 2][b].kill()
+        board.Cell[a][b + 1].kill()
+        board.Cell[a + 1][b + 1].kill()
+        board.Cell[a + 2][b + 1].birth(config.Square)
```

Comment:
Make a function which draws a preset from a 2d Boolean matrix, and then fill your config file with presets. For example, if you pass 
```
[[True,False,False],
[False,True,False],
[False,False,True]]
```
in the function you would get
```
x__
_x_
__x
```

Reply:
done